proof_of_concept/ddm_site.py

```diff
@@ -22,7 +22,7 @@
 class Site:
```

Comment:
Both in `register_site` and `SiteDescription` the store, runner, namespace, and policy_server are optional. But not for a Site object. I guess this is a choice that for now we just want to test with sites that have everything?

Reply:
Yes, eventually we may make a `Site` with optional components, but for now each site has everything for convenience.